code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -901,45 +902,91 @@
-		/datum/brain_trauma/mild/tourettes = 5
+		/datum/brain_trauma/severe/split_personality = 2, //Gotta remove those enemies to nanotrasen.
+		/datum/brain_trauma/special/imaginary_friend = 2,
+		/datum/brain_trauma/mild/tourettes = 2
 	)
 	dosage_traumas = list(
 		/datum/brain_trauma/severe/pacifism = 25
 	)
-	withdrawal_traumas = list()
+	withdrawal_traumas = list(
+		/datum/brain_trauma/mild/tourettes = 20
+	)
+	conflicting_reagent = /datum/reagent/mental/trisyndicotin
+	messagedelay = 30
+	var/is_affected = 0
+
+/datum/reagent/mental/hextrasenil/overdose(var/mob/living/carbon/M, var/alien, var/removed, var/scale)
+	. = ..()
+	if(!is_affected && prob(5))
+		var/mob/living/carbon/human/H = M
+		if(istype(H))
+			var/datum/antagonist/antag_data = get_antag_data(H.mind.special_role)
+			if(antag_data && (antag_data.flags & ANTAG_IMPLANT_IMMUNE))
+				H << "You feel the corporate tendrils of [current_map.company_name] try to invade your mind!"
+			else
+				clear_antag_roles(H.mind, 1)
+				H << "<span class='notice'>You feel a surge of loyalty towards [current_map.company_name]! Any negative intentions towards the company have been long forgotten.</span>"
+		is_affected = 1
+
+/datum/reagent/mental/hextrasenil/affect_conflicting(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagent/conflicting)
+	var/amount = min(removed, conflicting.volume)
+	holder.remove_reagent(conflicting.id, amount)
+	M.adjustBrainLoss(amount * 0.25)
 
 /datum/reagent/mental/trisyndicotin
 	name = "Trisyndicotin"
 	id = "trisyndicotin"
 	description = "Trisyndicotin is a super-strength, expensive antipsychotic medication intended for the use in interigation. Side effects include undying hatred to NanoTrasen and disrespect for authority."
 	reagent_state = LIQUID
 	color = "#888888"
-	metabolism = 0.03
+	metabolism = 0.02 * REM
 	data = 0
 	taste_description = "freedom"
 	goodmessage = list("You distrust Nanotrasen and their people.","You feel woke.","You have urges to speak out against NanoTrasen.","You feel the need to complain about NanoTrasen on the web.","You feel like things should be better.")
 	badmessage = list() //Actual Freedom.
 	worstmessage = list() //Actual Freedom.
 	suppress_traumas  = list(
-		/datum/brain_trauma/severe/pacifism = 10
+		/datum/brain_trauma/severe/pacifism = 5
 	)
+	conflicting_reagent = /datum/reagent/mental/hextrasenil
+	messagedelay = 30
+	var/is_affected = 0
+
+/datum/reagent/mental/trisyndicotin/overdose(var/mob/living/carbon/M, var/alien, var/removed, var/scale)
+	. = ..()
+	if(!is_affected && prob(5))
+		var/mob/living/carbon/human/H = M
+		if(istype(H))
+			for(var/obj/item/weapon/implant/loyalty/I in H.contents)
+				for(var/obj/item/organ/external/organs in H.organs)
+					if(I in organs.implants)
+						M.adjustBrainLoss(5)
+						H.adjustBruteLoss(10)
+						qdel(I)
+						break
+			H << "<span class='notice'><font size =3><B>Your loyalty implant has been deactivated. Freedom, at last!</B></font></span>"
```

Comment:
`to_chat()`.

Reply:
Also, how the fuck is this reagent deactivating a loyalty implant?

Reply:
In the loyalty implant description, the way loyalty implants work is that the implant sends nanobots into the bloodstream to modify the implated's brain. It even says that it only works if there are nanobots present in the blood system.

I was thinking of making the traitor medicine to be purchasable only, and contain pills of their own nanobots that interfere with implant.